File: app/src/main/java/com/aprivate/sean/ohcg/CurrentHandState.java
package com.aprivate.sean.ohcg;

import java.io.Serializable;
import java.util.HashMap;

public class CurrentHandState implements Serializable {

    HashMap<Integer, ScoreBoardItem> playerOrderMapToIndex;

    public HashMap<Integer, ScoreBoardItem> getPlayerOrderMapToIndex() {
        return playerOrderMapToIndex;
    }

    public void setPlayerOrderMapToIndex(HashMap<Integer, ScoreBoardItem> playerOrderMapToIndex) {
        this.playerOrderMapToIndex = playerOrderMapToIndex;
    }
    private int currentOrderNumber = 1;

    public int getTotalBidsForCurrentHand() {
        return totalBidsForCurrentHand;
    }

    public void setTotalBidsForCurrentHand(int totalBidsForCurrentHand) {
        this.totalBidsForCurrentHand = totalBidsForCurrentHand;
    }

    private int totalBidsForCurrentHand;

    public int getCurrentOrderNumber() {
        return currentOrderNumber;
    }

    public void setCurrentOrderNumber(int currentOrderNumber) {
        this.currentOrderNumber = currentOrderNumber;
    }
}

File: app/src/main/java/com/aprivate/sean/ohcg/CurrentHandState.kt
package com.aprivate.sean.ohcg

import java.io.Serializable
import java.util.*

class CurrentHandState : Serializable {
    var uid: UUID = UUID.randomUUID()
    var currentOrderNumber = 1
    var dealCount: Int = 0
    var totalBidsForCurrentHand: Int = 0

    val playerOrderMapToIndex: MutableMap<Int, ScoreBoardItem> = mutableMapOf()
}
